src/install.sh

```diff
@@ -1166,28 +1199,58 @@ function set_lb_type() {
   cat <<EOF
 
   *****************************************************************************
-  * When the Armory Platform runs it uses two load balancers. One for the API *
-  * gateway and one to serve the Web UI. Depending on how your network is     *
-  * configured, you will want these load balancers to either be 'internal' or *
-  * 'external'. After installation, it is also recommended that you configure *
+  * When the Armory Platform runs it exposes one loadbalancer to users.       *
+  * Depending on how your network is configured, you will want these load     *
+  * balancers to either be 'internal', 'external' or a 'clusterIP'.  For      *
+  * clusterIP deployments it uses 'kubectl' to create a tunnel to the         *
+  * cluster. After installation, it is also recommended that you configure    *
   * a firewall rule or security group to only allow access to whitelisted IPs.*
   *****************************************************************************
 
 EOF
   echo "Load balancer types: "
-  options=("Internal" "External")
+  options=("Internal" "External" "ClusterIP")
   PS3='Select the LB type you want to use: '
   select opt in "${options[@]}"
   do
     case $opt in
-        "Internal"|"External")
+        "Internal"|"External"|"ClusterIP")
             export LB_TYPE="$opt"
             echo "Using LB type: $opt"
             break
             ;;
         *) echo "Invalid option";;
     esac
   done
+
+  if [[ "${LB_TYPE}" == "ClusterIP" ]]; then
+    export SERVICE_TYPE=$LB_TYPE
+  else
+    export SERVICE_TYPE="LoadBalancer"
+  fi
+
+  save_response SERVICE_TYPE
+  save_response LB_TYPE
+}
+
+function save_response() {
+  echo "export ${1}=${!1}" >> $CONTINUE_FILE
+}
+
+function continue_env() {
+    if [[ ! -z  $NOPROMPT ]]; then
+      return
+    fi
+
+    if [[ -f $CONTINUE_FILE ]]; then
+        get_var "Found continue file at $CONTINUE_FILE from previous run, would you like to use it? (y/n): " USE_CONTINUE_FILE
```

Comment:
I feel like the default should be `y`, since we always prompt after `describe_installer`

Reply:
maybe store that `describe_installer` was already prompted and skip the requirement to hit "enter"